spec/terminal-tab-spec.js

```diff
@@ -51,4 +59,92 @@ describe('TerminalTab', () => {
     });
 
   });
+
+  describe('when the terminal:focus event is triggered', () => {
+    it ('navigates to an open terminal', () => {
+      terminalOpenAndWait(workspaceElement);
+      let bottomDock = atom.workspace.getBottomDock();
```

Comment:
bottomDock is now used fairly often in these  tests. I was considering moving it to beforeEach.